### PR TITLE
Replace extract-text-webpack-plugin with mini-css-extract-plugin

### PR DESCRIPTION
**Pull request by KKoukiou**
The former plugin is deprecated in favor of mini-css-extract-plugin.
See https://github.com/webpack-contrib/extract-text-webpack-plugin#usage

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -398,11 +398,15 @@ module.exports = {
             },
             {
                 test: /\.css$/,
-                loader: extract.extract("css-loader?minimize=&root=" + libdir)
```
Comment (KKoukiou) [inline webpack.config.js]: @martinpitt I don't understand what the 'root=" + libdir' was used for; can you give me a heads up?

File: package.json [hunk 2]
```diff
@@ -58,13 +58,13 @@
...
     "eslint-plugin-standard": "^4.0.1",
     "extend": "~3.0.0",
-    "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "html-webpack-plugin": "^3.2.0",
     "htmlparser": "~1.7.7",
     "imports-loader": "~0.6.5",
     "jed": "~1.1.0",
     "less": "~3.0.1",
     "less-loader": "~4.0.6",
+    "mini-css-extract-plugin": "0.8.0",
```
Comment (martinpitt) [inline package.json]: I think we should use ^ here. Not all our devDependencies use that (particularly not the older ones), but over time this is the most sensible default.

File: webpack.config.js (hunk 1 above)
Comment (martinpitt) [reply webpack.config.js]: This was introduced in commit 904ef8662288e26328f5f6f49d5a3e1b7aff3cea three years ago. I'm afraid I can't interpret this either, but if it builds and works, then your change is fine :-) Can you please force-push and trigger tests?

**Comment by martinpitt** [comment]
FTR, I diffed disk usage of dist/* before and after, and they are essentially identical.

**Pull request merged**